packages/components/filters/src/filter-menu/filter-menu.tsx

```diff
@@ -80,7 +107,29 @@ const menuStyles = css`
   position: relative;
   z-index: 5;
 `;
+
+const menuBodyStyle = css`
+  width: 100%;
+`;
+
 function FilterMenu(props: TFilterMenuProps) {
+  const menuBodyRef = useRef<HTMLDivElement>(null);
+
+  const focusMenuBody = useCallback(
+    (e) => {
+      if (menuBodyRef.current) {
+        const firstFocusableElementInMenuBody = findFirstFocusable(
+          menuBodyRef.current
+        );
+        if (firstFocusableElementInMenuBody) {
```

Comment:
If there's a focusable element in the body, then `preventDefault` to prevent radix from putting focus on a focusable item in the header, and focus on the element in the body.

If there is no focusable element, then do nothing and let radix handle focus management